README.md

```diff
@@ -40,9 +40,9 @@ Features
 - Cores available for RetroArch/Libretro and OpenEmu.
 - Community-provided translations for several languages via [Weblate](https://hosted.weblate.org/engage/mgba).
 - Many, many smaller things.
-
 #### Game Boy mappers
 
+
```

Comment:
Looks like the newline got transposed by mistake.

Reply:
Sorry. Will fix this.